networkit/cpp/centrality/test/CentralityGTest.cpp

```diff
@@ -1735,6 +1857,25 @@ TEST_P(CentralityGTest, testDynTopHarmonicCloseness) {
     }
 }
 
+TEST_F(CentralityGTest, testApproxSpanningEdgeAndWriteVectors) {
+    Aux::Random::setSeed(42, false);
+
+    Graph G = ErdosRenyiGenerator(100, 0.1, false).generate();
+    G.indexEdges();
+    constexpr double eps = 0.1;
+
+    ApproxSpanningEdge apx(G, eps);
+    apx.run();
+    SpanningEdgeCentrality se(G, eps);
+    se.runApproximationAndWriteVectors("./resApprox.result");
```

Comment:
Better also check for existence of the written file (see `IOGTest.cpp` for details). Also, since this is function is likely rarely used, read the created file and do sanity checks (number of lines as expected, same is approximation vector from object, ...).